aptos-move/framework/supra-framework/sources/supra_governance.move

```diff
@@ -252,6 +348,39 @@ module supra_framework::supra_governance {
         );
     }
 
+    public fun update_supra_governance_config(
+        supra_framework: &signer,
+        voting_duration_secs: u64,
+        min_voting_threshold: u64,
+        voters: vector<address>,
+    ) acquires SupraGovernanceConfig, SupraGovernanceEvents {
+        system_addresses::assert_supra_framework(supra_framework);
+
+        let supra_governance_config = borrow_global_mut<SupraGovernanceConfig>(@supra_framework);
+        supra_governance_config.voting_duration_secs = voting_duration_secs;
+        supra_governance_config.min_voting_threshold = min_voting_threshold;
```

Comment:
`assert!(vector::length(&voter) >= min_voting_threshold && min_voting_threshold > vector::length(&voter)/2 && min_voting_threshold > 2, ERROR)`
These checks are needed during upgrade
Also `voting_duration` should not be too small so put an assertion that it is at least 1 hour.

Reply:
Make sure these checks are present also in `initialize`